README.md

```diff
@@ -6,9 +6,11 @@ The IBM Power Virtual Systems Container Storage Interface (CSI) Driver provides
 
 
 # CSI Specification Compatibility Matrix
-| PowerVS CSI Driver \ CSI Version | v1.5.0 |
-| ----------------------------- | -------|
-| main branch | yes |
+| PowerVS CSI Driver | Kubernetes | CSI | Golang |
+| ----------------------------- | ----------- | -------- | -------- |
+| main | 1.24 | 1.5.0 | 1.18 |
+| 0.2.0 | 1.24 | 1.5.0 | 1.18 |
```

Comment:
This is not true yet, lets hold this for now till we update required modules.

/hold

Reply:
/unhold

Reply:
modules are updated to expected versions.